playwright/tests/integration/properties/button.spec.ts

```diff
@@ -7,23 +7,23 @@ test('default', async ({ page }) => {
   await editor.inscription.expectHeader('Button');
   const properties = editor.inscription.section('Properties');
   const section = properties.collapsible('General');
-  const name = section.input({ label: 'Name' });
-  const action = section.input({ label: 'Action' });
+  const name = section.badge({ label: 'Name' });
+  const action = section.badge({ label: 'Action' });
```

Comment:
this sounds wrong to me...
the tester shouldn't need to care if there is a badge component over the input.
The badge stuff should be included in the input page object.
so if you want to write to an input then the page object first then should click into the input so the badge goes away.